ui/utilities/common-newport/_index.scss

```diff
@@ -688,4 +688,14 @@ form {
   .ng-valid span.nds-required {
     color: $color-valid-control;
   }
-}
\ No newline at end of file
+}
+
+.via-nds .nds-form--multi-input__container .nds-tooltip_container .nds-nonfocused_control .nds-is-absolute .nds-popover.nds-popover_tooltip.nds-nubbin_bottom,
+.via-nds .nds-lookup .nds-tooltip_container .nds-nonfocused_control .nds-is-absolute .nds-popover.nds-popover_tooltip.nds-nubbin_bottom,
+.via-nds .nds-typeahead>div.nds-grid .nds-tooltip_container .nds-nonfocused_control .nds-is-absolute .nds-popover.nds-popover_tooltip.nds-nubbin_bottom {
+  left: calc(0% - 30px);
```

Comment:
One other thing @dskay3 is that we should write `left` as `#{$left}` so that we can make it automatically flip when we compile the RTL stylesheets (Same goes for the right rule below).

Reply:
Gotcha, will do!